db/migrate/20160810093024_unify_character_encoding_of_all_tables.rb

```diff
@@ -2,7 +2,7 @@ class UnifyCharacterEncodingOfAllTables < ActiveRecord::Migration
   # SQL provided by David Harper
 
   def up
-    psd_only = ENV['PSD_ONLY']
+    psd_only = ENV.fetch('PSD_ONLY', nil)
```

Comment:
I presume this migration has already been run on prod? If so, then I don't think we should change the code, since it will no longer reflect what was run. If it's because of rubocop, we could add a rubocop disable thing for this file?